gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/graphml/GraphMLReader.java

```diff
@@ -343,6 +344,7 @@ public static Builder build() {
         private String vertexLabelKey = GraphMLTokens.LABEL_V;
         private boolean strict = true;
         private long batchSize = 10000;
+        private XMLInputFactory inputFactory = XMLInputFactory.newInstance();
```

Comment:
It is wasteful to always instantiate a default inputFactory considering it may not be used. Alternatively I can write in create method like:

```
        public GraphMLReader create() {
            if (this.inputFactory == null) {
                this.inputFactory = XMLInputFactory.newInstance();
            }
            return new GraphMLReader(this);
        }
```